django/http/response.py

```diff
@@ -299,10 +299,15 @@ def __init__(self, content=b'', *args, **kwargs):
         self.content = content
 
     def __repr__(self):
-        return '<%(cls)s status_code=%(status_code)d, "%(content_type)s">' % {
+        if self.get('Content-Type'):
```

Comment:
Local variable approach seems bad to me, there's no need in a variable, used only once.
Why do you prefer `in` lookup?